components/dashboard/cards/testimonial.tsx

```diff
@@ -0,0 +1,67 @@
+import { useDispatch } from 'react-redux'
+import { AppDispatch } from '@/lib/redux/store'
+import { setTestimonials, setDisplay } from '@/lib/redux/slices/testimonials'
+import { StyledTestimonialCard, StyledTestimonialInput } from './StyledCards'
+import { Close } from '@mui/icons-material'
+import { Button } from '@mui/material'
+import { useState } from 'react'
+import { createTestimonials } from '@/lib/db/cms/testimonials'
+
+const TestimonialCard = ({ field }: any) => {
+    const dispatch = useDispatch<AppDispatch>()
+    const [isFlipped, setIsFlipped] = useState(false)
+    const { display, email, id, name, rating, review } = field
+
+    const handleDisplayChange = (event: {
+        target: { id: string; checked: boolean }
+    }) => {
+        const { id, checked } = event.target
+        dispatch(setDisplay({ id, checked }))
+    }
+
+    const toggleDeletionView = event => {
+        setIsFlipped(!isFlipped)
+    }
+    return (
+        <StyledTestimonialCard>
+            <div className={isFlipped ? 'card is-flipped' : 'card'} id={id}>
+                <div className="card-face front">
+                    <Close className="close" onClick={toggleDeletionView} />
+                    <div className="header">
+                        <h3>{name}</h3>
+                        <span>{rating}/5</span>
+                    </div>
+                    {review}
+                    <footer>
+                        <StyledTestimonialInput
+                            type="checkbox"
+                            className="tgl tgl-flip"
+                            id={id}
+                            onChange={handleDisplayChange}
+                            checked={!!display}
```

Comment:
FYI, if `display` is a boolean, you do not need to `!!` it. Booleans will already render down to truthy/falsey without the !!